core/primitives-core/src/version.rs

```diff
@@ -138,6 +138,12 @@ pub enum ProtocolFeature {
     /// Increases the number of chunk producers.
     TestnetFewerBlockProducers,
     /// Enables stateless validation which is introduced in <https://github.com/near/NEPs/pull/509>
+    /// LowerValidatorKickoutPercentForDebugging: lower block and chunk validator kickout percent from 90 to 50.
+    /// SingleShardTracking: single shard tracking for stateless validation.
+    /// StateWitnessSizeLimit: state witness size limits.
+    /// PerReceiptHardStorageProofLimit: limit the size of storage proof generated by a single receipt.
+    /// WitnessTransactionLimits: size limits for transactions included in a ChunkStateWitness.
+    /// NoChunkOnlyProducers: no chunk-only producers in stateless validation.
     StatelessValidationV0,
```

Comment:
nit: I would be tempted to rename it to just `StatelessValidation`. I doubt we'll have a `StatelessValidationV2` (or V1?) even if so we'll probably have a cool new name for it :)

Reply:
+1